### DISCUSSION TITLE
How to change chart data range ?

### DISCUSSION THREAD

**Discussion by clmtkrz** [PHPOffice/PhpSpreadsheet#2837] [Q&A] [answered]
Hi everyone,

I'm actually wondering about how could I modify a chart already in my excel file. I want modify through PHPSPreadSheet the data range, by example (I have multiples series so it is complicating the objective) :

=Evolution!$C$37:$C$40
to 
=Evolution!$C$37:$C$400

Any idea (I didn't found anything in the docs/or samples).
Thanks for reading
Respectfully

**Comment by oleibman** [comment, marked as answer]
```php
$dataSeriesValues->setDataSource('Evolution!$C$37:$C$400');
```
